src/index.js

```diff
@@ -5,13 +5,22 @@ import { Provider as ReduxProvider } from "react-redux";
 import "./index.css";
 import "./assets/icomoon.css";
 import App from "./App";
+import SandboxApp from './sandbox/SandboxApp';
 import configureStore from "./Redux/Store";
 
 const store = configureStore();
 
+const renderApp = () => {
+  if(process.env.REACT_APP_SANDBOX) {
+    return <SandboxApp />;
+  }
+
+  return <App />;
+};
+
 ReactDOM.render(
   <ReduxProvider store={store}>
-    <App />
+    {renderApp()}
```

Comment:
It could be a functional component rather than function.. JSX will look a bit cleaner.
<RenderApp />

Reply:
Addressed in [commit](https://github.com/singnet/refactored-dapp/pull/110/commits/097a133d21db9ab5d2cf7ef85c28a0673cc92d6b)